amr-wind/utilities/IOManager.H

```diff
@@ -77,6 +77,8 @@ public:
 
     bool is_restart() const { return !m_restart_file.empty(); }
 
+    const amrex::Vector<Field*> plot_fields() const { return m_plt_fields; }
+
```

Comment:
```suggestion
    const amrex::Vector<Field*>& plot_fields() const { return m_plt_fields; }
```

Don't create a copy of the vector